sql/parser/eval.go

```diff
@@ -27,6 +27,7 @@ import (
 	"time"
 
 	"github.com/cockroachdb/cockroach/proto"
+	"github.com/cockroachdb/cockroach/sql/driver"
```

Comment:
I think it is a bit weird for `sql/parser` to depend on `sql/driver`. Sharing the date/timestamp formats is not worth it in my opinion. And note that `sql/parser` should really be supporting a wider variety of formats than `sql/driver` needs for output. Actually, I don't think `sql/driver` needs any formats. It should just use `time.Time.String()`. I'm guessing the formatting is only used for the logic tests.